dartagnan/src/test/java/com/dat3m/dartagnan/litmus/LitmusVulkanFairLivenessTest.java

```diff
@@ -0,0 +1,43 @@
+package com.dat3m.dartagnan.litmus;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.configuration.Property;
+import com.dat3m.dartagnan.utils.Result;
+import com.dat3m.dartagnan.utils.rules.Provider;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.sosy_lab.common.configuration.Configuration;
+import static com.dat3m.dartagnan.configuration.OptionNames.PROGRESSMODEL;
+
+import java.io.IOException;
+import java.util.EnumSet;
+
+@RunWith(Parameterized.class)
+public class LitmusVulkanFairLivenessTest extends AbstractLitmusTest {
+
+    public LitmusVulkanFairLivenessTest(String path, Result expected) {
+        super(path, expected);
+    }
+
+    @Parameterized.Parameters(name = "{index}: {0}, {1}")
+    public static Iterable<Object[]> data() throws IOException {
+        return buildLitmusTests("litmus/VULKAN/", "VULKAN-Liveness-Fair");
+    }
+
+    @Override
+    protected Provider<Configuration> getConfigurationProvider() {
+        return Provider.fromSupplier(() -> Configuration.builder()
+                .setOption(PROGRESSMODEL, "fair")
```

Comment:
Can we use the enum type instead of the string in case we ever change the names?

Reply:
Yes, I already changed this and the configuration issue like a minute after opening the PR.
I just didn't push yet because of the issues with the litmus tests.